src/lib/y2partitioner/clients/main.rb

```diff
@@ -6,13 +6,24 @@
 Yast.import "Stage"
 Yast.import "Wizard"
 
+# Work around YARD inability to link across repos/gems:
+# (declaring macros here works because YARD sorts by filename size(!))
+
+# @!macro [new] AW
+#   @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FAbstractWidget:${0}
+# @!macro [new] CW
+#   @see http://www.rubydoc.info/github/yast/yast-yast2/CWM%2FCustomWidget:${0}
```

Comment:
interesting, in the end I like it. It is quite simple. Just maybe use a longer and more descriptive name when reading code only. so maybe AbstractWiget and CustomWidget?